app/routes/interestgroups/components/InterestGroupDetail.js

```diff
@@ -123,10 +123,11 @@ type Props = {
 
 function InterestGroupDetail(props: Props) {
   const { group } = props;
+  const canEdit = group.actionGrant && group.actionGrant.includes('edit');
```

Comment:
If group exists, I believe actionGrant is always an array - so we probably dont need this short-circuit check?